src/In.ProjectEKA.HipService/Link/LinkController.cs

```diff
@@ -127,5 +130,24 @@ public async Task LinkPatientCareContextFor(LinkPatientRequest request)
                 Log.Error(exception, exception.StackTrace);
             }
         }
+        
+        [HttpPost(PATH_ON_LINK_ADD_CONTEXTS)]
+        public AcceptedResult LinkFor(LinkCareContextResultRequest request)
+        {
+            Log.Information("Link Care Context Result Request received." +
+                            $" RequestId:{request.RequestId}, " +
+                            $" Timestamp:{request.Timestamp},");
+            if (request.Error != null)
+            {
+                Log.Information($" Error Code:{request.Error.Code}," +
+                                $" Error Message:{request.Error.Message},");
+            }
+            else if (request.Acknowledgement != null)
+            {
+                Log.Information($" Acknowledgement Status:{request.Acknowledgement.Status},");
+            }
+            Log.Information($" Resp RequestId:{request.Resp.RequestId}");
+            return Accepted();
```

Comment:
Don't use this Log, use what's their CareContextDiscoveryController/PatientController.cs